test/unit/services/query-service.test.js

```diff
@@ -41,14 +80,106 @@ describe('query-service', () => {
         }
       ])
 
-      const { response, summariesMode } = await fetchAnswer({}, input, chatHistory, false, false)
+      const { response, summariesMode, hallucinated } = await fetchAnswer({}, input, chatHistory, false, false)
 
       expect(mockInvoke).toHaveBeenCalledWith({ chat_history: chatHistory, input })
       expect(summariesMode).toBe(false)
+      expect(hallucinated).toBe(false)
       expect(response).toStrictEqual(JSON.stringify(answer))
       expect(createStuffDocumentsChain).toHaveBeenCalledWith(expect.objectContaining({ prompt }))
     })
 
+    test('validateResponseLinks to validate a correctly structured response without hallucinated links', async () => {
```

Comment:
Merge mistake? These two `validateResponseLinks` tests have been moved to test/unit/utils/validators.test.js